logic/love_count.py

```diff
@@ -37,14 +38,15 @@ def top_lovers_and_lovees(utc_week_start, dept=None, limit=20):
         if c.received_count == 0:
             continue
         employee_key = c.key.parent()
-        if dept:
-            employee = employee_key.get()
-            if employee.meta_department == dept or employee.department == dept:
-                lovees.append((employee_key, c.received_count))
-        else:
+        employee = employee_key.get()
+        if (
```

Comment:
Not blocking but I think this could need a comment saying what condition we are looking for to append en employee to the list.